docs/Discussions

```diff
@@ -0,0 +1,12 @@
+## DevOps Mutualization SIG Discussions
+
+DevOps Mutualization SIG uses Github Discussions that are only visible to members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/) as agreed upon in the [meeting on Feb 18th 2020](https://github.com/finos/devops-mutualization/issues/18)
```

Comment:
`Github Discussions` **should be** `[Github Team Discussions](https://odp.finos.org/docs/project-collaboration#github-team-discussions)`

Reply:
`members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/)` **should be** `members who are part of the [devops-mutualization-participants team](https://github.com/orgs/finos/teams/devops-mutualization-participants/)`